prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+let person = {
+  firstName: 'Bob',
+  lastName: 'Ross',
+  hobby: 'art',
+};
+
+console.log(person);
+
+let fullName = person.firstName + ' ' + person.lastName;
+console.log("The person's name is: " + fullName);
+
+person.job = 'artist';
+console.log("The person's current job is: " + person.job);
+
+person.previousJob = 'pizza boy';
```

Comment:
Bracket notation is requested per the instructions
`person['previousJob'] = 'pizza boy';`